test/uk/gov/hmrc/tai/service/CodingComponentServiceSpec.scala

```diff
@@ -71,16 +65,12 @@ class CodingComponentServiceSpec extends PlaySpec with MockitoSugar {
 
   private val codingComponentList = Seq[CodingComponent](
     CodingComponent(PersonalAllowancePA, Some(123), 12345, "some description"),
-    CodingComponent(CarFuelBenefit, Some(124), 66666, "some other description"),
+    CodingComponent(CarFuelBenefit, Some(124), 6666, "some other description"),
```

Comment:
Is this an intentional change? Passes with both 66666 and 6666?

Reply:
Oops, I'll undo that